examples/user_interaction/server/server.js

```diff
@@ -72,16 +72,25 @@ function handleRequest(request, response) {
 
     // Necessary headers because the Node.js and React dev servers run in different ports.
     response.setHeader('Access-Control-Allow-Origin', '*');
-    response.setHeader('Access-Control-Allow-Methods', 'GET, POST, OPTIONS, PUT, PATCH, DELETE');
-    response.setHeader('Access-Control-Allow-Headers', 'X-Requested-With,content-type');
+    response.setHeader('Access-Control-Allow-Methods', 'GET, OPTIONS');
+    response.setHeader('Access-Control-Allow-Headers', 'traceparent');
     response.setHeader('Access-Control-Allow-Credentials', true);
 
     let result = '';
     let code = 200;
+    // Logic related to CORS pre-flight requests.
```

Comment:
Maybe say "Accept all CORS pre-flight requests"? Would it make sense to move the `response.setHeader` calls into this block of code too?

Reply:
Moved only the `setHeader`s related to the CORS request.